### DISCUSSION TITLE
Mouse hover Trail effect

### DISCUSSION THREAD

**Discussion by EpiQuip** [matteobruni/tsparticles#1586] [Q&A] [answered]
Hello! I was wondering if there is a way to use mouse hover trail effect with the specified coordinates (without mouse interaction)? I want to use this perfect effect in my application with the moving `div` container. 

Thank you very much!

**Comment by matteobruni** [comment, marked as answer]
Something like this? https://codepen.io/matteobruni/pen/OJpQvdO

**Reply by EpiQuip** [reply]
Not exactly, what I meant, but I think I got your point - I just need to use emitters! Thank you very much!
